### PR TITLE
Allow metadata & lang properties embed-element

### PR DESCRIPTION
**Pull request by havardholvik**
We have forgotten to forward the non-bool attributes to the iframe. This change forwards both `lang` and `metadata`

### CODE REVIEW

File: embed/tester/tester.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import "https://cdn.srv.whereby.com/embed/v1.js";
+import "../whereby-embed.js";
```
Comment (havardholvik) [inline embed/tester/tester.js:1]: Unsure if there are any downsides to use the non-CDN version of the embed element in the tester. This change makes it at least way easier to develop locally, as you need to manually change this and remember to change it back before commit.

Comment (martrik) [reply embed/tester/tester.js:1]: I guess it should be OK?

**Review by martrik — APPROVED** [review]
🍰

File: embed/tester/tester.js (hunk 1 above)
Comment (havardholvik) [reply embed/tester/tester.js:1]: I think so too. There might be an argument for the GitHub hosted and CDN-hosted becoming out of sync, but that should be a minor one if we get a proper pipeline up and running.

**Pull request merged**